slide-deck.js

```diff
@@ -391,22 +456,33 @@ class slideDeck extends HTMLElement {
     }
   }
 
+  #hidePartsChange = () => {
+    this.slideNotes.forEach((note) => {
+      note.toggleAttribute('hidden', this.hideParts === 'note');
+    });
+
+    this.slideCanvas.forEach((canvas) => {
```

Comment:
Because `this.slideCanvas` is assigned before slide setup is called, the contents of `this.slideCanvas` is only the elements the user applied the attribute to, and not all of the ones that web component applied the attribute to in `#setupSlides()`. 

This means that when you hide canvas, you get this-
<img width="1409" alt="image" src="https://github.com/oddbird/slide-deck/assets/167908/daf33e68-60f2-4e13-8723-f56fd139a4b9">

Only the slides with notes have their canvases hidden.

Reply:
Yeah, this was intentional, but maybe not the right choice. For slides without separate canvas and notes, hiding the canvas would mean _hiding the entire container_. That doesn't seem like exactly what we want.